### ISSUE TITLE
Feature request - computed attributes in map

### ISSUE THREAD

**Issue by cop98** [traccar/traccar-web#1058]
It would be very useful to be able to display the computed attributes in the pop-up in the map as well.
Currently (traccar 5.5) it is only possible to display the standard attributes of Traccar

**Comment by tananaev** [comment]
Wrong. You can display custom attributes as well.

**Comment by tananaev** [comment]
Maybe the issue with the space. Not sure if that's supported.

**Comment by cop98** [comment]
I tried with an attribute without spaces but the result is the same.
<img width="1066" alt="Screenshot 2022-12-08 alle 20 23 47" src="https://user-images.githubusercontent.com/49091679/206551442-771c3bfc-aff3-416c-ab64-9a800d6949bd.png">

<img width="429" alt="Screenshot 2022-12-08 alle 20 24 28" src="https://user-images.githubusercontent.com/49091679/206551457-4a0811c8-05ab-4a42-b293-cc52439bec19.png">



If I try to enter the name and save anyway the following error appears:


<img width="709" alt="Screenshot 2022-12-08 alle 20 25 03" src="https://user-images.githubusercontent.com/49091679/206551235-6cb6212a-6a3a-4e2c-a587-0c7b70e38757.png">

**Comment by tananaev** [comment]
The error is completely unrelated. It means you have too much data stored in your user attributes.

**Comment by cop98** [comment]
But beyond the error. Is it normal that I can't search for "LitersFuel"?

**Comment by tananaev** [comment]
Obviously if it's a custom attribute, you won't be able to search for it, but you should still be able to add it.

**Comment by cop98** [comment]
OK, I was able to insert the attribute into the list.
But when I go to click on the vehicle in the map, this error appears.
Do you know what it means?

Thanks

![image](https://user-images.githubusercontent.com/49091679/206557226-e4a2ca46-9afa-4c81-8b4d-b20f427b272b.png)

**Comment by tananaev** [comment]
Probably means something wrong with the attribute, but impossible to tell what exactly based on this error message.

**Comment by cop98** [comment]
Hello,
I made a video where you can see the error. can you figure out what's wrong?

This does this with all attributes calculated and with all users.

The attribute is correct, in the detail it shows correctly, it is only in the map that it does not show up

Thank you very much

https://www.youtube.com/watch?v=vfcuSm6U7UY&ab_channel=TheSupernik98

**Comment by tananaev** [comment]
It doesn't really help. What's the value of the attribute? Check the database.

**Comment by cop98** [comment]
The value of the attribute is 0

**Comment by tananaev** [comment]
Can you please provide full JSON from the database.

**Comment by cop98** [comment]
I don't know what you're talking about. If you have a guide of how to do this, I will gladly do it.

If it would be helpful, I have shared with you the DB

Thank you

https://drive.google.com/file/d/14sTV9flGVdnoDsCR_Ovv9UcU5JkNNfnI/view?usp=sharing

**Comment by cop98** [comment]
Unfortunately, I don't know how to get what you say.
Can this report be useful?
[report.xlsx](https://github.com/traccar/traccar-web/files/10199568/report.xlsx)

**Comment by tananaev** [comment]
I checked the last message and it doesn't have the attribute in question, so the value is not zero as you said. Going forward I ask you to provide clear evidence for your statements.

**Comment by cop98** [comment]
sorry, I had changed the attribute to exam but actually didn't write it down.
I used the attribute "DeepSleep" as well as "carburante finto" as I had shown in the video since you had told me that maybe spaces were not supported.
The error is the same
in the last message in the report above is = 0 (DeepSleep=0).

I will leave you some screens taken from the video posted above


<img width="390" alt="Screenshot 2022-12-10 alle 17 59 03" src="https://user-images.githubusercontent.com/49091679/206866595-3d03efa0-f612-4716-8e45-78c6086fb440.png">
<img width="381" alt="Screenshot 2022-12-10 alle 17 59 20" src="https://user-images.githubusercontent.com/49091679/206866614-76624f37-5aaa-445d-b0e5-bbada9b83047.png">
<img width="1208" alt="Screenshot 2022-12-10 alle 17 59 42" src="https://user-images.githubusercontent.com/49091679/206866616-8332b3f1-64ba-4852-baaf-a79d7bfeb090.png">
